src/dmd/cpreprocess.d

```diff
@@ -65,6 +66,21 @@ FileName preprocess(FileName csrcfile)
               there. To get the headers and sppn.exe, dmc will need to be installed on the test machines:
               http://ftp.digitalmars.com/Digital_Mars_C++/Patch/dm857c.zip
          */
+        if (target.objectFormat() == Target.ObjectFormat.coff)
```

Comment:
The comment a couple of lines above is now obsolete.

Reply:
I'll add sppn support after this is pulled.